core/consensus/grandpa/has_change.hpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CONSENSUS_GRANDPA_HAS_CHANGE_HPP
+#define KAGOME_CONSENSUS_GRANDPA_HAS_CHANGE_HPP
+
+#include "primitives/block_header.hpp"
+
+namespace kagome::consensus::grandpa {
+  struct HasChange {
+    HasChange(const primitives::BlockHeader &block) {
+      for (auto &digest : block.digest) {
+        auto digest2 = boost::get<primitives::Consensus>(&digest);
```

Comment:
The new naming scheme doesn't look great for me. 
Names like 2,3,4 imply different contents (not formats).
In my view, the old approach with digest_opt or digest_decoded suits better here.

Reply:
Absolutely.